react-frontend/src/components/StyleComponents/bannerSimple.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+
+export const BannerSubTitle = styled.p.attrs({
+  className: 'bannerSubTitle',
+})`
+  color: #313132;
+  font-size: 22px;
+  position: relative;
+  @media only screen and (max-width: 800px) {
+    font-size: 10pt;
+    text-align: justify;
+    font-size: 16px;
+    text-align: center;
+    width: 100%;
+  }
+`;
```

Comment:
for **`bannerSubTitle`**, to match @singhinderdeep 's Figma designs, I believe you can delete the `@media` query, and replace this block with:

```
   color: #313132;
   font-size: 22px;
   font-weight: 700;
   text-align: center;
```